core-blocks/verse/index.js

```diff
@@ -29,6 +34,9 @@ export const settings = {
 			source: 'children',
 			selector: 'pre',
 		},
+		align: {
```

Comment:
Hi @nfmohit, thank you for your contribution!
The changes look good from the code point of view.
I'm not sure if other people agree with this thought, but in my option, this attribute should be named textAlign. Align attribute is used by the block align hook (supports: align) and maybe in the future we want to add block alignment here so using textAlign attribute we would have the door open to that :)

Reply:
Thank you for reviewing this @jorgefilipecosta! I've added [a new commit](https://github.com/WordPress/gutenberg/pull/6700/commits/bb887bcda43780254c7b86eacaa1e880d3ad6bfb) which changes `align` to `textAlign`.

I can see the other core blocks like header, subhead, paragraph etc. blocks are still using the `align` attribute for text alignment. Do you think they should be updated to `textAlign` for consistency? Let me know if I should open a new issue for that and submit a PR.

Thank you!